bash_command_line_102/bash_command_line_102.md

```diff
@@ -482,12 +482,12 @@ rm brown_bear.txt
 <div class = "warning">
 <b style="color: rgb(var(--color-highlight));">Warning!</b><br>
 
-Removing files from your command line interface with the `rm` command **removes** them from your computer. It does not move them to your "Trash" folder.
+Removing files via your command line interface with the `rm` command **removes** them from your computer. It does not move them to your "Trash" folder.
 
-Be completely sure that you want to remove a file before using the `rm` command.
+Be completely sure that you want to remove a file before using the `rm` command as YOU WILL NOT BE ABLE TO GET REMOVED FILES BACK unless you have another copy backed up somewhere else.  
```

Comment:
```suggestion
Be completely sure that you no longer need a file before using the `rm` command as YOU WILL NOT BE ABLE TO GET REMOVED FILES BACK unless you have another copy backed up somewhere else.  
```
framing to focus on the necessary condition that would lead one to want use rm command rather than just "want to remove a file" to more concretely drive home what it means to "remove a file"